### ISSUE TITLE
Children take All Available Beds/Can't Hire Wanderers

### ISSUE THREAD

**Issue by desagas** [ldtteam/minecolonies#6871]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.6.5.36.1.2
- MineColonies Version: minecolonies-0.14.84-ALPHA-universal.jar
- Structurize Version: structurize-0.13.160-ALPHA-universal.jar
- Related Mods and their Versions: ATM61.5.8b

## Expected behavior

If children being born is turned on in townhall, I should have time to hire or move citizens around before they immediately take up the population cap. If I build a new house, I should be able to hire the travellers who come to my tavern before children arrive. Perhaps there should be (maybe I need to make a feature request) a "child expected" countdown in the town hall, or a chat message that says a child will be arrive in "X time here." I guess the bug is there seems to be no balance between travellers and children.

## Actual behavior

Within minutes of building new houses, or any structure that adds beds (tavern, upgraded town hall, houses, etc), children fill the beds, which means I do not have time to even hire wanderers before I can not hire them because a child has magically appeared out of nowhere.

Now, I know that you can turn kids being born off, but, I would like to have a good mix of both kids and travellers being admitted to my colony.

## Steps to reproduce the issue

1. 
2. 
3. 

### Logs

- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by ravenbuilder934** [comment]
So the [feature request](https://github.com/ldtteam/minecolonies-features/issues/356) is for an announcement when a kid will be born soon, and this is to increase the delay?

**Comment by desagas** [comment]
Exactly. To announce that a kid will be born soon, and add a delay so that they are not born at the exact same time as the notification. Perhaps a day later or whatever happens to work best.

**Comment by desagas** [comment]
Either that, or make kids that show up to your colony not automatically join the colony?

**Comment by Raycoms** [comment]
Announcement doesn't work, because you can't annoucne and then not spawn one when there is no space, and homeless children is also not a good idea at all. We already have a small timeout to get born, is that not enough in most cases?
